cmd/bnsd/x/username/handlers.go

```diff
@@ -18,9 +18,9 @@ func RegisterRoutes(r weave.Registry, auth x.Authenticator) {
 	r = migration.SchemaMigratingRegistry("username", r)
 
 	b := NewTokenBucket()
-	r.Handle(RegisterTokenMsg{}.Path(), &registerTokenHandler{auth: auth, bucket: b})
-	r.Handle(TransferTokenMsg{}.Path(), &transferTokenHandler{auth: auth, bucket: b})
-	r.Handle(ChangeTokenTargetsMsg{}.Path(), &changeTokenTargetsHandler{auth: auth, bucket: b})
+	r.Handle(&RegisterTokenMsg{}, &registerTokenHandler{auth: auth, bucket: b})
```

Comment:
Yeah, nicer.
You could also do `(*RegisterTokenMsg)(nil)` if you want to get really carried away, but I think this is more readable